### PR TITLE
SSO redirect logging, first/last name in session

### PR DESCRIPTION
**Pull request by pkarman**
This PR does 2 things:

* adds more logging for SSO redirect logic
* captures the SAML first/last name attributes as separate fields

### CODE REVIEW

File: app/models/css_authentication_session.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class CssAuthenticationSession
   include ActiveModel::Model
   include ActiveModel::Serializers::JSON
 
-  attr_accessor :id, :name, :roles, :station_id
+  attr_accessor :id, :name, :roles, :station_id, :first_name, :last_name
```
Comment (pkarman) [inline app/models/css_authentication_session.rb]: possibly useful to https://github.com/department-of-veterans-affairs/caseflow/issues/12113

Comment (lomky) [reply app/models/css_authentication_session.rb]: If the name previously was just built out of "$first $last", wouldn't that imply the suffix is coming through how we're grabbing last here?

Comment (pkarman) [reply app/models/css_authentication_session.rb]: if first name is `John E.` and last name is `Smith, Jr` then presumably it's easier to sort on the last name value if you don't need to parse spaces with a regex?

Comment (lomky) [reply app/models/css_authentication_session.rb]: good point; it does grab us a valid start of the last name!

**Pull request merged**